### DISCUSSION TITLE
where is the ? Alt apk installer.exe file ?

### DISCUSSION THREAD

**Discussion by drishyadipshali** [mjishnu/alt-app-installer#62] [General]
We are not getting altapkinstaller.exe file. May I know what is the processes to use this application .

**Comment by mjishnu** [comment]
just download altappinstaller.exe from [release](https://github.com/mjishnu/alt-app-installer/releases/tag/v2.6.9) and run it here is the video guide https://youtu.be/ayIilTc-6u4
